### ISSUE TITLE
postgis doesn't install address_standardizer

### ISSUE THREAD

**Issue by dgobaud** [Homebrew/homebrew#49129]
Installed postgis successfully and CREATE EXTENSION postgis; works but below do not

CREATE EXTENSION postgis_sfcgal;

ERROR: could not open extension control file "/usr/local/Cellar/postgresql/9.4.4/share/postgresql/extension/postgis_sfcgal.control": No such file or directory

CREATE EXTENSION address_standardizer;

ERROR:  could not open extension control file "/usr/local/Cellar/postgresql/9.4.4/share/postgresql/extension/address_standardizer.control": No such file or directory

CREATE EXTENSION address_standardizer_data_us;

ERROR: could not open extension control file "/usr/local/Cellar/postgresql/9.4.4/share/postgresql/extension/address_standardizer_data_us.control": No such file or directory

**Comment by ilovezfs** [comment]
You need to build postgis from source and then it will be there. With the bottle, it won't be.

```
bash-3.2$ find /usr/local/share/postgresql/extension|grep addr
/usr/local/share/postgresql/extension/address_standardizer--1.0--2.2.1.sql
/usr/local/share/postgresql/extension/address_standardizer--2.2.1--2.2.1next.sql
/usr/local/share/postgresql/extension/address_standardizer--2.2.1.sql
/usr/local/share/postgresql/extension/address_standardizer--2.2.1next--2.2.1.sql
/usr/local/share/postgresql/extension/address_standardizer.control
/usr/local/share/postgresql/extension/address_standardizer.sql
/usr/local/share/postgresql/extension/address_standardizer_data_us--2.2.1--2.2.1next.sql
/usr/local/share/postgresql/extension/address_standardizer_data_us--2.2.1.sql
/usr/local/share/postgresql/extension/address_standardizer_data_us--2.2.1next--2.2.1.sql
/usr/local/share/postgresql/extension/address_standardizer_data_us.control
/usr/local/share/postgresql/extension/address_standardizer_data_us.sql
```

```
brew install --build-from-source  postgis
```

**Comment by DomT4** [comment]
> You need to build postgis from source and then it will be there. With the bottle, it won't be.

Can we fix that?

**Comment by ilovezfs** [comment]
@DomT4 yeah it's actually a missing dependency. Not sure which yet, but will soon.

**Comment by ilovezfs** [comment]
@DomT4 Fixed: #49136
